BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -504,107 +506,148 @@ - (void)cardNumberErrorString:(NSString*)errorString {
 
 - (void)tokenizeCard {
     [self.view endEditing:YES];
-    
+
+    if (self.cardCapabilities != nil && self.cardCapabilities.isUnionPay && self.cardCapabilities.isSupported) {
+        [self enrollCard];
+    } else {
+        [self basicTokenization];
+    }
+}
+
+- (void)basicTokenization {
+    __block BTCardRequest *cardRequest = self.cardRequest;
```

Comment:
Do these need to be `__block`? They don't appear to be referred to inside the tokenizeCard block.